config/initializers/good_job.rb

```diff
@@ -3,7 +3,7 @@
 
   config.good_job.preserve_job_records = true
   config.cleanup_preserved_jobs_before_seconds_ago = 604_800 # 1 semaine
-  config.good_job.on_thread_error = ->(exception) { Sentry.capture_exception(exception) }
+  config.good_job.on_thread_error = ->(exception) { Sentry.capture_exception(exception) } # this is never called !
```

Comment:
je pense que je supprimerais cette ligne dans une prochaine PR, just to be safe

Reply:
Pour être sûr qu'on ne l'appelle pas, est-ce qu'on changerais pas en `Sentry.capture_exception("This should never happen")` avec l'exception en contexte ?

Reply:
bonne idée ! je fais ça dans une prochaine PR , je me le note